astropy/visualization/units.py

```diff
@@ -19,13 +19,19 @@ def quantity_support(format="latex_inline"):
       >>> from astropy import units as u
       >>> from astropy import visualization
       >>> with visualization.quantity_support():
-      ...     plt.figure()
+      ...     fig = plt.figure()
       ...     plt.plot([1, 2, 3] * u.m)
       [...]
       ...     plt.plot([101, 125, 150] * u.cm)
       [...]
+      ...     fig.axes[0].yaxis.set_units(u.km)
       ...     plt.draw()
```

Comment:
Given @pllim's comment, this should be changed to:

```suggestion
      ...     fig, ax = plt.subplots()
      ...     ax.plot([1, 2, 3] * u.m)
      [...]
      ...     ax.plot([101, 125, 150] * u.cm)
      [...]
      ...     ax.yaxis.set_units(u.km)
      ...     plt.draw()
```

Reply:
Thanks. I've changed it.